backend/compact-connect/lambdas/python/common/cc_common/data_model/data_client.py

```
@@ -1231,3 +1359,267 @@ def encumber_license(self, adverse_action: AdverseActionData) -> None:
+            if target_adverse_action.effectiveLiftDate is not None:
+                raise CCInvalidRequestException('Encumbrance has already been lifted')
+
+            # Check if this is the last remaining unlifted adverse action
+            unlifted_adverse_actions = [
+                aa
+                for aa in adverse_action_records
+                if aa.effectiveLiftDate is None and str(aa.adverseActionId) != adverse_action_id
+            ]
+
+            # Get the privilege record
+            license_type_name = LicenseUtility.get_license_type_by_abbreviation(compact, license_type_abbreviation).name
```

Pylint (3.3.7)

[convention] Line too long (120/100)

(C0301)